### ISSUE TITLE
Implement a library to create a connection between the app the MySQL

### ISSUE THREAD

**Issue by Omeramsc** [Code-Collaborate-Community/tiger#108]
Three options: 

- **flask-mySQLdb**  : throws an error when trying to pip install: "python setup.py egg_info" failed with error code 1" 

- **SQLAlchemy** : work well, but maybe a bit too complicated for our scope. 

- **Python MySQL** : It seems easy and simple. (https://www.w3schools.com/python/python_mysql_getstarted.asp)

Let's decide today and implement it so all teams will be able to begin working on their issues @BarakMerdler @Nomasson @Yarinh55

**Comment by Omeramsc** [comment]
I succeeded in installing flask-mysql, but it goes with installing: gcc, python3-devel, mysql-devel and wheel. 
I don't know why, I just know that if any of those are missing, the install fail. 

We should check the benefits of each library and make a decision.

**Comment by BarakMerdler** [comment]
@Omeramsc I don't understand what the problem is.
The Python MySQL is working fine? So why we need to check for another library?
What is the benefit of changing? I started to work with that, and it works great.

**Comment by Omeramsc** [comment]
@BarakMerdler I don't mind either way, but since we don't yet have a PR, It can't hurt to check what other tools we have at our disposal. I've been researching the different options for about three weeks now, It's not something I came up with ATM. 
I found a lot of guides about flask_mysqldb regarding creating sessions, different checks, etc, so I thought it might be useful to use it since it's quite popular and built upon an improved forked version of mysql-connector (Python MySQL).
I don't think the choice is THAT relevant to the scope of this project, but we do need to implement something and issue a PR because It's blocking most of our team's issues.
